### PR TITLE
Clean up README. Remove non-rule rule. PROFIT!!!

### PR DESCRIPTION
**Pull request by mathisto**
## If this PR were a GIF, what would it be?
Clean up. Clean up

## Description of change(s)
- "All roads lead to Rome" was removed from the rule list since it wasn't a rule.
- Moved "WeAreGo.md" 'rule' to the list of links. This was also not a rule.
- Rule three needed to be swapped...OBVIOUSLY.
- Rule three needed to propose five, and then suddenly, at the last moment, remember to be three.
- Added example of `1.` ordered list syntax to rule 4.
- Added a link to "Mastering Markdown" on rule 4.
- Rephrased wording on rule 5 and added links to Brian and I's GH profiles.

## Diligence checklist (omit non-applicable items):
- [x] Has Readme/Wiki/documentation been updated?
- [x] Are you a lumberjack?
- [x] Is that OK?
- [x] Is this code production ready?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
 ## Rules
 Hereth be yon rules:
 1. Thou shalt ask ONLY ridiculous questions, and ridiculous shall thine questions be.
 1. All Pull Requests must be accompanied by an animated gif. No exceptions.
-1. All ordered lists must use one-dot GH flavored markdown syntax
-1. three sir!
-1. Edit [WeAreGo.md](docs/WeAreGo.md) checking the box next to your name to prove your karate & friendship. (**test if you can commit & push**)
-1. All roads lead to Rome
-1. Get your hands dirty, try for thirty - After that, ask for Matt
+1. Five...no three sir!
```
Comment (fiteclub) [inline README.md:17]: - [x] [That's Numberwang!](https://youtu.be/0obMRztklqU)

File: README.md [hunk 2]
```diff
@@ -6,17 +6,16 @@
...
+1. All ordered lists must use `1.` [GitHub flavored markdown](https://guides.github.com/features/mastering-markdown/) syntax. ex: `1. First Item` `1. Second Item` `1. Third Item`
```
Comment (fiteclub) [inline README.md:18]: LGTM!

File: README.md (hunk 1 above)
Comment (fiteclub) [reply README.md:17]: Do these conversations have to be resolved before it can be approved/merged?

Comment (mathisto) [reply README.md:17]: No. Not at all, but it's good form.

**Review by fiteclub — APPROVED** [review]
👍 Production ready

**Review by dmohar — APPROVED** [review]

**Pull request merged**